builder/metadata.go

```diff
@@ -149,6 +151,60 @@ func (b *Builder) getUpstreamFormatRange(version string) (format string, first,
 	return format, first, latest, err
 }
 
+// UpdateFormatFile update the format number in the full-chroot format file
+func (b *Builder) UpdateFormatFile(version int) error {
+	formatFile := filepath.Join(b.Config.Builder.ServerStateDir, "image", b.MixVer, "full/usr/share/defaults/swupd/format")
+	if _, err := os.Stat(formatFile); err == nil {
```

Comment:
I think all the old issues are still here. I'll hold off on reviewing.

Reply:
Yes it's not updated, just rebased to builder breakage.